doc/en/components/grids/_shared/advanced-filtering.md

```diff
@@ -57,6 +57,17 @@ constructor() {
 }
 ```
 <!-- end: WebComponents -->
+<!-- React -->
+```html
+<{ComponentSelector} data={this.nwindData} autoGenerate="false" ref={this.gridRef} allowAdvancedFiltering="true">
+    <IgrGridToolbar>
+        <IgrGridToolbarActions>
+            <IgrGridToolbarAdvancedFilering></IgrGridToolbarAdvancedFilering>
```

Comment:
`<IgrGridToolbarAdvancedFilering></IgrGridToolbarAdvancedFilering>` That doesn't seem to be required, it's not included for the snippets for blazor, wc or angular.